src/OSPSuite.Core/Domain/Quantity.cs

```diff
@@ -43,7 +43,7 @@ public interface IQuantity : IFormulaUsable, IUsingFormula, IWithValueOrigin
       /// <summary>
       ///    The value in the displayed unit
       /// </summary>
-      bool TryGetValueInDisplayUnit(out double result);
+      (double value, bool success) TryGetValueInDisplayUnit();
```

Comment:
This is a better API. Since it was not used before, I have no problem changing it